pkgs/top-level/all-packages.nix

```diff
@@ -8974,6 +8974,8 @@ with pkgs;
 
   gtdialog = callPackage ../development/libraries/gtdialog { };
 
+  gtkclipblock = callPackage ../tools/misc/gtkclipblock { stdenv = gcc13Stdenv; };
```

Comment:
any reason to use gcc13 specifically?

Reply:
It doesn't build with `stdenv` and `gcc12Stdenv`, so my hypothesis is that it requires some feature introduced in GCC 13 (though I haven't dug too deep into this).

Reply:
With the next staging-next merge, this should become the default.